### PR TITLE
Change the unit naming for byte sizes and make it unambigious

### PR DESCRIPTION
**Pull request by janhack**

### CODE REVIEW

**Review by sergeyshemyakov — APPROVED** [review]
Looks good!

File: proposals/lip-0002.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ As the Lisk protocol evolves and gains popularity, a clear increase in usage and
 
 ## Rationale
 
-We propose to set a maximum block size of 15KB (kilobytes). This size will maintain the yearly blockchain growth below 50GB, even when blocks are always at full capacity, and the active delegates do not miss any blocks. The summary of the numerical implications are as follows:
+We propose to set a maximum block size of 15 KiB (15 * 1024 bytes). This size will maintain the yearly blockchain growth below 50 GiB, even when blocks are always at full capacity, and the active delegates do not miss any blocks. The summary of the numerical implications are as follows:
 
-* Maximum block size: 15KB.
-* Blockchain size will grow a **maximum** of 47.3GB per year.
+* Maximum block size: 15 KiB.
+* Blockchain size will grow a **maximum** of 47.3 GiB per year.
```
Comment (AndreasKendziorra) [inline proposals/lip-0002.md:30]: If I'm not wrong, it must be 45.1 if the unit is GiB, as 1 GiB = 1024^3 bytes.
And we have a maximum of `(6*60*24*365)*15*1024 bytes = 48439296000 bytes` per year.

BTW: Do we need to incorporate leap years? :)

File: proposals/lip-0002.md [hunk 2]
```diff
@@ -112,17 +112,17 @@ We believe that given these numbers, 15KB is the most adequate choice in terms o
...
 
-5. Update `constants.maxPayloadLength` to 15KB.
+5. Update `constants.maxPayloadLength` to 15 KiB.
 
 6. Remove `constants.maxTxsPerBlock` from `constants.js`.
 
 7. Remove the redundant `numberOfTransactions` check from blocks verification for historical data, as it is already included in block's signature.
 
-8. Remove the redundant `payloadLength` check from blocks verification for historical data (up to 1048576KB), as it is already included in block's signature.
+8. Remove the redundant `payloadLength` check from blocks verification for historical data (up to 1048576 KiB), as it is already included in block's signature.
```
Comment (AndreasKendziorra) [inline proposals/lip-0002.md:123]: Here I'm not sure if changing the unit without changing the value is correct. I actually can't recall right now where the value comes from.

**Review by AndreasKendziorra — CHANGES_REQUESTED** [review]

File: proposals/lip-0002.md (hunk 2 above)
Comment (janhack) [reply proposals/lip-0002.md:123]: I just remove this value, as it is about removing the check and the value is not important for the specifications.

File: proposals/lip-0002.md (hunk 1 above)
Comment (janhack) [reply proposals/lip-0002.md:30]: I just changed the unit name and was assuming the specifications consistently used 1024 as base. I updated this line.

File: proposals/lip-0002.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ As the Lisk protocol evolves and gains popularity, a clear increase in usage and
 We propose to set a maximum block size of 15 KiB (15 * 1024 bytes). This size will maintain the yearly blockchain growth below 50 GiB, even when blocks are always at full capacity, and the active delegates do not miss any blocks. The summary of the numerical implications are as follows:
 
 * Maximum block size: 15 KiB.
-* Blockchain size will grow a **maximum** of 47.3 GiB per year.
+* The block history will grow around 45 GiB per year (approximately 6*60*24*365.25*15/(1024*1024) GiB).
```
Comment (AndreasKendziorra) [inline proposals/lip-0002.md:30]: But "around" instead of "maximum" is rather misleading as we have very few transactions right now. Don't you think?

**Review by AndreasKendziorra — CHANGES_REQUESTED** [review]

File: proposals/lip-0002.md (hunk 3 above)
Comment (janhack) [reply proposals/lip-0002.md]: I added "assuming full blocks" to the end of the sentence.

**Review by sergeyshemyakov — APPROVED** [review]
Looks good!

**Review by karmacoma — APPROVED** [review]

**Pull request merged**